src/Files.App/Files.App.csproj

```diff
@@ -101,7 +101,9 @@
     </None>
     <None Update="Assets\terminal\terminal.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
+    </None>    
+    <Content Include="Resources\AppCenterKey.txt" />
+    <Content Include="Resources\BingMapsKey.txt" />    
     <None Update="Resources\PreviewPanePropertiesInformation.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Could you check if CopyToOutputDirectory is also needed? I had a problem with Themes working when deploying locally but not when building the package due to missing this.

Reply:
I guess we won't know until it's packaged, I'll generate a build of this branch to test with.

Reply:
It worked as intended 👍